drivers/wifi/simplelink/simplelink_support.c

```diff
@@ -65,6 +65,44 @@ struct sl_connect_state sl_conn;
 /* Network Coprocessor state, including role and connection state: */
 static struct nwp_status nwp;
 
+#if defined(CONFIG_NET_IPV4) && defined(CONFIG_NET_CONFIG_MY_IPV4_ADDR)
+/* Convert IP addr string into an array of integers */
+static int tokenize_ipv4addr(u8_t *ip_array, char const *ip_string)
```

Comment:
You could use `net_ipaddr_parse()` function here that will do basically the same.

Reply:
> You could use net_ipaddr_parse() function here that will do basically the same.

I guess net_addr_pton() might be even better fit.

Reply:
@jukkar @pfalcon I have updated the code to use net_addr_pton(). It is a good idea.